python/xorbits/datasets/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022-2023 XProbe Inc.
+# derived from copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .adapter import MARS_DATASET_CALLABLES, MARS_DATASET_TYPE, _install
+from .backends.huggingface.core import from_huggingface
+
+globals().update(MARS_DATASET_CALLABLES)
```

Comment:
Why need this line?

Reply:
I need to update the wrapped callables to datasets module. e.g. a wrapped `from_huggingface`.